tests/Phinx/Migration/AbstractMigrationTest.php

```diff
@@ -190,7 +190,33 @@ public function testInsert()
         $table = new Table('testdb', [], $adapterStub);
 
         $migrationStub->setAdapter($adapterStub);
-        $migrationStub->insert($table, ['row' => 'value']);
+        @$migrationStub->insert($table, ['row' => 'value']);
+    }
+
+    public function testInsertString()
+    {
+        // stub migration
+        $migrationStub = $this->getMockForAbstractClass('\Phinx\Migration\AbstractMigration', ['mockenv', 0]);
+
+        // stub adapter
+        $adapterStub = $this->getMockBuilder('\Phinx\Db\Adapter\PdoAdapter')
+            ->setConstructorArgs([[]])
+            ->getMock();
+        $adapterStub->expects($this->once())
+            ->method('bulkinsert');
+
+        $migrationStub->setAdapter($adapterStub);
+        @$migrationStub->insert('testdb', ['row' => 'value']);
+    }
+
+    /**
+     * @expectedException \PHPUnit\Framework\Error\Deprecated
```

Comment:
Seems like this class does not exist for older phpunit :(

I'm ok with not having this test, as it does not provide a lot of value to us

Reply:
Yeah, had to move this check inside of the test to detect PHP version to use proper class name. Works now.